src/azure-cli/azure/cli/command_modules/config/_help.py

```diff
@@ -9,13 +9,14 @@
 helps['config'] = """
 type: group
 short-summary: Manage Azure CLI configuration.
+long-summary: Available since Azure CLI 2.10.0.
```

Comment:
The docs are working on generating "available in version" information automatically.